pandas/tests/groupby/test_whitelist.py

```diff
@@ -164,33 +164,43 @@ def raw_frame():
 @pytest.mark.parametrize('axis', [0, 1])
```

Comment:
remove this and it will just use the axis fixture; note that axis will take on [0, 1, 'index', 'columns'], so you may have to adjust some of the test conditions